bitcoin/examples/ecdsa-psbt-simple.rs

```diff
@@ -0,0 +1,246 @@
+//! Implements a simple multi-input PSBT signing example
+//!
+//! The purpose of this section is to construct a PSBT that
+//! spends multiple inputs and signs it.
+//! We'll cover the following
+//! [BIP 174](https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki)
+//! roles:
+//!
+//! - **Creator**: Creates a PSBT with multiple inputs and outputs.
+//! - **Updater**: Adds Witness and SegWit V0 data to the PSBT.
+//! - **Signer**: Signs the PSBT.
+//! - **Finalizer**: Finalizes the PSBT.
```

Comment:
I don't think we should include the finalizer role in this example, if people want to finalize they should use `rust-miniscript`. It is not possible to finalize using only `rust-bitcoin`.

Reply:
Are you sure?
This is exactly what does https://github.com/rust-bitcoin/rust-bitcoin/blob/594eb18728fb8a433b1045f7e9c9c507ec54349b/bitcoin/examples/ecdsa-psbt.rs#L226-L248

I agree that we can add a warning that using `PsbtExt` in `rust-miniscript` is the way to go for production-level stuff.

Reply:
hmm, I'd lean towards thinking we should remove `finalize_psbt` from that example also.

Reply:
Can we move this to an issue?
If yes, I can move forward with this PR and we can decide what to do with the Finalizer part in the PSBT examples in the issue.

Reply:
@tcharding in some limited scenarios you can finalize directly with rust-bitcoin. Like if you know that all your inputs have some simple form (e.g. are all p2wpkh).

I think we should move this to a separate issue/discussion about whether we should say "always use rust-miniscript" or should provide examples/documentation for people to try to make ad-hoc finalizers like this.

Reply:
Works for me! Thanks